py3status/module.py

```diff
@@ -219,6 +252,9 @@ def run(self):
                         result['instance'] = self.module_inst
                         result['name'] = self.module_name
 
+                    # set universal module options in result
+                    result.update(self.module_options)
```

Comment:
They are not global, you can specify them for each module individually. For example:
```
time {
    ...
    separator = false
}
```